runner/app/routes/image_to_video.py

```diff
@@ -17,6 +19,35 @@
 
 logger = logging.getLogger(__name__)
 
+
+def handle_pipeline_error(e: Exception) -> JSONResponse:
+    """Handles exceptions raised during image-to-video pipeline processing.
+
+    Args:
+        e: The exception raised during image-to-video processing.
+
+    Returns:
+        A JSONResponse with the appropriate error message and status code.
+    """
+    if isinstance(e, torch.cuda.OutOfMemoryError):
```

Comment:
I see this part of the code is repeated in every pipeline, would it maybe make sense to extract it and reuse?

Reply:
@leszko as also stated in the discord I implemented a new variant of @gioelecerati's logic in ae8df749e107cf2e13ab8fb86f5a884675370e1c.